server/src/resolvers/Blog.ts

```diff
@@ -1,6 +1,8 @@
+import { PrismaClient } from "@prisma/client";
+
 export const Blog = {
 
-    author: async ({ authorId }, args, { prisma }) => {
+    author: async ({ authorId }:{ authorId: string}, args: any, { prisma }: { prisma: PrismaClient }) => {
```

Comment:
I see that in a lot of resolvers your only parameter is `id`.

A hint: create a type like:

```
export type ObjectIdProps = {
  id: string
}
```

You can do it in e.g. a `types.ts` file where you put the types you want to **reuse**. Then, you just say

```
author: async ({ id }: ObjectIdProps, args: any, { prisma }: { prisma: PrismaClient }) => {
```

And in the next files, where you require only an ID, you can use this type. Then you don't create so much redundant code.